nemo/collections/asr/parts/utils/offline_clustering.py

```diff
@@ -486,7 +486,7 @@ def getMultiScaleCosAffinityMatrix(
     timestamps_in_scales: List[torch.Tensor],
     device: torch.device = torch.device('cpu'),
 ) -> torch.Tensor:
-    """
+      """
```

Comment:
Did you make this tab intentionally? seems like it is causing style error in the test.

Reply:
Fixed this on my own.